### DISCUSSION TITLE
Does anyone have a dockerfile that works with nextjs 14?

### DISCUSSION THREAD

**Discussion by rgarcia1129** [vercel/community#6049] [Help] [answered]
### Summary

Im able to run the next app in a docker container but for some reason i cant reach it using the urls provided upon startup. When hitting both the localhost and network urls, it either hangs or I get any error saying "didnt send any data"

### Example

https://stackoverflow.com/questions/78117966/why-is-my-nextjs-14-app-not-responding-when-ran-in-docker-container

### Steps to Reproduce

I would be ok with even the boilerplate code being dockerized as long as I could hit it. Once I can get a basic dockerfile working I can adapt it to my actual repo. just wondering if any one has ran into this or has examples I could reference. The one in the nextjs examples repo is a much older version of next.

**Comment by leerob** [comment, marked as answer]
You can use this example! https://github.com/vercel/next.js/tree/canary/examples/with-docker
